chef_master/source/install_supermarket.rst

```diff
@@ -296,9 +295,9 @@ Install Supermarket Directly (without a cookbook)
 =====================================================
 While there are many benefits to using the cookbook method to install Supermarket, there are also cases where it's simpler to set up the Supermarket installation manually. These steps will walk you through the process of manually configuring your private Supermarket server.
 
-Before following these steps, be sure to complete the OAuth setup process detailed in the `Chef Identity </install_supermarket.html#chef-identity>`__ section of this guide.  
+Before following these steps, be sure to complete the OAuth setup process detailed in the `Chef Identity </install_supermarket.html#chef-identity>`__ section of this guide.
```

Comment:
Consider:  " #. First, complete the OAuth setup process detailed in `Chef Identity </install_supermarket.html#chef-identity>`__ "